From 19c66a954cd9a0d8eabfb66b9dd02483d417cfeb Mon Sep 17 00:00:00 2001
From: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>
Date: Tue, 9 Nov 2021 15:25:44 +0200
Subject: [PATCH 4/5] HACK: Allow DomD enumerate PCI devices

Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>
---
 drivers/xen/pci.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/xen/pci.c b/drivers/xen/pci.c
index 2c890f4f2cbc..5fbb8f40dc04 100644
--- a/drivers/xen/pci.c
+++ b/drivers/xen/pci.c
@@ -206,8 +206,10 @@ static struct notifier_block device_nb = {
 
 static int __init register_xen_pci_notifier(void)
 {
+#if 0
 	if (!xen_initial_domain())
 		return 0;
+#endif
 
 	return bus_register_notifier(&pci_bus_type, &device_nb);
 }
-- 
2.38.1

